From f88f362a8ee3b0edb126e3b3db5dbb9c0c71da53 Mon Sep 17 00:00:00 2001
From: Wu Guoxing <b02248@freescale.com>
Date: Thu, 14 Aug 2008 16:15:06 +0800
Subject: [PATCH] ENGR00083952 The first frame will show one strap on the top

The first frame will show one strap on the top,
this is because the sensor need some time
to setup, so, add some delay.

Signed-off-by: Guoxing Wu <b02248@freescale.com>
---
 drivers/media/video/mxc/capture/ov2640.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/media/video/mxc/capture/ov2640.c b/drivers/media/video/mxc/capture/ov2640.c
index c6b5de0..8d237da 100644
--- a/drivers/media/video/mxc/capture/ov2640.c
+++ b/drivers/media/video/mxc/capture/ov2640.c
@@ -385,6 +385,8 @@ static sensor_interface *ov2640_config(int *frame_rate, int high_quality)
 	else
 		ov2640_init_mode(ov2640_mode_800_600);
 
+	msleep(300);
+
 	return interface_param;
 }
 
-- 
1.5.4.4

